core/new-gui/src/app/workspace/component/navigation/navigation.component.html

```diff
@@ -49,22 +49,23 @@
         <button (click)="onClickRestoreZoomOffsetDefault()" nz-button title="reset zoom">
           <i nz-icon nzType="fullscreen" nzTheme="outline"></i>
         </button>
-        <button nz-button [disabled]="isDownloadDisabled()" [nzDisabled]="isDownloadDisabled()" [nzDropdownMenu]="menu"
-                id="downloadExcelOptions" nz-dropdown title="download execution result">
+        <button [disabled]="workflowResultExportService.isResultExportDisabled()" [nzDropdownMenu]="menu"
```

Comment:
please make this a local variable instead of calling a function, this function will be called in every change detection cycle, even if the change is completely irrlevent

Reply:
done